pkg/web/static/js/envelope.js

```diff
@@ -97,17 +97,8 @@ document.body.addEventListener('htmx:configRequest', (e) => {
   }
 });
 
-// Use human readable identifier types in the transaction preview.
 document.body.addEventListener('htmx:afterSettle', (e) => {
-  if (e.detail.requestConfig.path === '/v1/transactions/prepare' && e.detail.requestConfig.verb === 'post') {
```

Comment:
This was moved to a different file. The only code change was to remove the if statement.